deploy.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
+set -e
+
+GITHUB_TAG=${GITHUB_REF/refs\/tags\//}
 
 docker login -u="$QUAY_USERNAME" -p="$QUAY_PASSWORD" quay.io
-docker tag keboola/ex-generic-mock-server quay.io/keboola/ex-generic-mock-server:$TRAVIS_TAG
+docker tag keboola/ex-generic-mock-server quay.io/keboola/ex-generic-mock-server:$GITHUB_TAG
```

Comment:
chcelo by to premigrovat na dockerhub, ... v 1P je heslo pre `keboolabot`, staci tam vytvorit novy image ... v Github Action su secrets dostupne automaticky (`DOCKER_LOGIN` ...).

Reply:
to dyžtak v dalším PR - potřebuju dotáhnout generic ex